locales/cs.json

```diff
@@ -0,0 +1,193 @@
+  "You can see how it works on ": "Můžete se podívat, jak to funguje ",
+  "You can choose to enable or disable this option.": "Tuto možnost můžete povolit nebo zakázat.",
+  "Enable analytics to help improve Boostnote": "Povolit analytiku pro další zlepšování Boostnote",
+  "Crowdfunding": "Crowdfunding",
+  "Dear Boostnote users,": "Vážení uživatelé Boostnote,",
+  "Thank you for using Boostnote!": "Děkujeme, že používáte Boostnote!",
+  "Boostnote is used in about 200 different countries and regions by an awesome community of developers.": "Boostnote je používán v přibližně 200 různých zemích a regionech úžasnou komunitou vývojářů.",
+  "To support our growing userbase, and satisfy community expectations,": "Abychom podpořili rostoucí uživatelskou základnu a naplnili očekávání komunity,",
+  "we would like to invest more time and resources in this project.": "rádi bychom do tohoto projektu investovali více času a zdrojů.",
+  "If you use Boostnote and see its potential, help us out by supporting the project on OpenCollective!": "Pokud používáte Boostnote a vidíte jeho potenciál, pomozte nám podporou projektu na OpenCollective!",
+  "Thanks,": "Děkujeme,",
+  "The Boostnote Team": "Tým Boostnote",
+  "Support via OpenCollective": "Podpořit přes OpenCollective",
+  "Language": "Jazyk",
+  "Default New Note": "Výchozí nová poznámka",
+  "English": "Angličtina",
+  "German": "Němčina",
+  "French": "Francouzština",
+  "Show \"Saved to Clipboard\" notification when copying": "Zobrazit při kopírování upozornění \"Uloženo do schránky\"",
+  "All Notes": "Všechny poznámky",
+  "Starred": "Starred",
+  "Are you sure to ": "Opravdu ",
+  " delete": " odstranit",
+  "this folder?": "tuto složku?",
+  "Confirm": "Potvrdit",
+  "Cancel": "Storno",
+  "Markdown Note": "Markdown poznámka",
+  "This format is for creating text documents. Checklists, code blocks and Latex blocks are available.": "Tento formát je pro tvorbu textových dokumentů. Zaškrtávací seznamy, bloky kódu a Latex bloky jsou dostupné.",
+  "Snippet Note": "Útržek kódu",
+  "This format is for creating code snippets. Multiple snippets can be grouped into a single note.": "Tento formát je pro tvorbu útržků kódu. Do jedné poznámky lze seskupit více útržků.",
+  "Tab to switch format": "Pro přepnutí formátu zmáčkněte Tab",
+  "Updated": "Aktualizováno",
+  "Created": "Vytvořeno",
+  "Alphabetically": "Abecedně",
+  "Counter": "Počítadlo",
+  "Default View": "Výchozí zobrazení",
+  "Compressed View": "Komprimované zobrazení",
+  "Search": "Vyhledat",
+  "Blog Type": "Typ blogu",
+  "Blog Address": "Adresa blogu",
+  "Save": "Uložit",
+  "Auth": "Autentizace",
+  "Authentication Method": "Autentizační metoda",
+  "JWT": "JWT",
+  "USER": "UŽIVATEL",
+  "Token": "Token",
+  "Storage": "Uložiště",
+  "Hotkeys": "Horké klávesy",
```

Comment:
Is `Horké klávesy` a thing in czech? 😄 I'm more used to `Klávesové zkratky` ('Shortcuts' translated to czech)